networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,88 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../networking/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-operator-release-notes-1-4"]
+== Network Observability Operator 1.4.0
+The following advisory is available for the Network Observability Operator 1.4.0:
+
+* link:https://errata.devel.redhat.com/advisory/2023:116729[RHSA-2023:1817 Network Observability Operator 1.4.0]
+
+[id="network-observability-channel-removal-1.4"]
+=== Channel removal
+You must switch your channel from `v1.0.x` to `stable` to receive the latest Operator updates. The `v1.0.x` channel is now removed.
+
+[id="network-observability-operator-1.4.0-features-enhancements"]
+=== New features and enhancements
+
+[id="network-observability-without-loki-1.4"]
+==== Network Observability without Loki
+The Network Observability Operator is now functional and usable without Loki. If Loki is not installed, it can only export flows to KAFKA or IPFIX format and provide metrics in the Network Observability metrics dashboards. For more information, see xref:../../networking/network_observability/installing-operators.adoc#network-observability-without-loki_network_observability[Network Observability without Loki]. 
+
+[id="network-observability-enhanced-configuration-and-ui-1.4"]
+==== Notable enhancements
+The 1.4 release of the Network Observability Operator adds improvements and new capabilities to the {product-title} web console plugin and the Operator configuration. 
+
+This release introduces the following web console enhancements:
```

Comment:
I was looking at the screen recording of the preview and you might want to consider making this sentence and the one about configuration enhancements a discrete heading or something like that. I feel like they kind of get buried in the text. It is totally up to you though.

Reply:
Ooo discrete heading for these is a great suggestion! TY!!